.github/workflows/print_versions.sh

```diff
@@ -10,6 +10,6 @@ git --version
 
 # This will fail if the build failed.
 grass --version
-grass --tmp-location XY --exec g.version -e
+grass --tmp-location XY --exec g.version -ecxrgb
```

Comment:
```suggestion
grass --tmp-location XY --exec g.version -ergb
```
No need for citation or copyright, both static info, only noise here.

Reply:
Yes, but it also highlighted that there was an encoding difference in windows, that didn't match the output here, where the accentuated characters are displaying correctly here, but not on the OSGeo4W package.

Reply:
I see. Maybe a specific test would be more fitting, but I have no specific suggestion.

Reply:
I'm rethinking about it, and I think the suggestion is alright. It would be more something to have in a manual test plan, as I don't see a way to have it tested automatically. If it was tested automatically, we would need to trust that the build environment doesn't have this problem also.
So we only need to collectively remember to check it before release, or something that shows that the encoding isn't wrong.

Reply:
> So we only need to collectively remember to check it before release, or something that shows that the encoding isn't wrong.

For collectively remembering stuff, it may go here:
https://github.com/OSGeo/grass/blob/main/doc/howto_release.md